kratos/geometries/line_2d_3.h

```diff
@@ -726,9 +729,23 @@ class Line2D3
     */
     SizeType EdgesNumber() const override
     {
-        return 2;
+        return 1;
     }
 
+    /**
+     * @brief This method gives you all edges of this geometry.
+     * @details This method will give you all the edges with one dimension less than this geometry.
+     * For example a triangle would return three lines as its edges or a tetrahedral would return four triangles as its edges but won't return its six edge lines by this method.
+     * @return GeometriesArrayType contains this geometry edges.
+     * @see EdgesNumber()
+     * @see Edge()
+     */
```

Comment:
```suggestion
    /// @copydoc Geometry::GenerateEdges
```

Same comment about `@copydoc`.

Reply:
Done. Thanks for your suggestion.